deps/rabbit/src/rabbit_backing_queue.erl

```diff
@@ -105,7 +105,7 @@
 
 %% Called to inform the BQ about messages which have reached the
 %% queue, but are not going to be further passed to BQ.
--callback discard(rabbit_types:msg_id(), pid(), state()) -> state().
+-callback discard(rabbit_types:basic_message(), pid(), state()) -> state().
```

Comment:
should this be `mc:state()` instead of `rabbit_types:basic_message()`?

Reply:
@noxdafox can you please see if you can refactor this to `mc:stat/0` and make it work for the deduplication plugin? `rabbit_types:basic_message/0` is AMQP 0-9-1-specific so we'd prefer to use the more generic `mc` module (which is AMQP 1.0-inspired but works for all protocols).

Thank you!

Reply:
Will do. Will submit another PR.

Reply:
Done in #13382